components-core/src/main/java/com/airwallex/android/core/Airwallex.kt

```diff
@@ -359,23 +360,70 @@ class Airwallex internal constructor(
     ) {
         when (session) {
             is AirwallexPaymentSession -> {
-                val paymentIntent = session.paymentIntent
-                confirmPaymentIntent(
-                    paymentIntentId = paymentIntent.id,
-                    clientSecret = requireNotNull(paymentIntent.clientSecret),
-                    paymentMethod = paymentMethod,
-                    cvc = cvc,
-                    currency = session.currency,
-                    customerId = paymentIntent.customerId,
-                    paymentConsentId = paymentConsentId,
-                    additionalInfo = additionalInfo,
-                    returnUrl = if (paymentMethod.type == PaymentMethodType.CARD.value)
-                        AirwallexPlugins.environment.threeDsReturnUrl()
-                    else session.returnUrl,
-                    autoCapture = session.autoCapture,
-                    flow = flow,
-                    listener = listener
-                )
+                if (paymentMethod.type == PaymentMethodType.GOOGLEPAY.value) {
```

Comment:
This method is pretty huge. Can we split this out into multiple methods? 

Also this is probably a stylistic choice but I feel like being explicit with if/else is easier to read than using `let` with `?:` as an else case.

Reply:
Good idea! extracted to the separate method `checkoutGooglePay`